auto_nag/scripts/not_landed.py

```diff
@@ -112,6 +147,10 @@ def check_splinter(self, attachment):
         if attachment['is_patch'] == 0 or attachment['is_obsolete'] == 1:
             return None
 
+        data = base64.b64decode(attachment['data']).decode('utf-8')
+        if data.startswith('https://github.com'):
+            return None
```

Comment:
Should we ignore MozReview patches too?

Reply:
We are already ignoring them as their content type is ''text/x-review-board-request''.